darkmoon/api/v1/metadata/views.py

```diff
@@ -54,25 +50,29 @@ async def list_metadata(
     ),
     length: int = Query(10, ge=1, le=500),
 ) -> list[MetadataEntity]:
-    """Return list of metadata that matches the parameters in the database.
+    """Get list of metadata that matches the parameters in the database.
 
     Parameters:
-        file_name (Optional[str]): The name of the file being
-            searched. Is None by default.
-        hash_type (Optional[str]): The type of hash. Is None by default.
-        hash (Optional[str]): Hash of the file. Is None by default.
+        collection (AsyncIOMotorCollection): The database collection to query.
+        file_name (str): The name of the file being searched.
+        hash (str): The hash of the file.
+        page (int): The page number to iterate to.
+        length (int): The number of items per page.
 
-    Returns:
-        documents (list[MetadataEntity]): List of all documents that match
-            parameters in the database
 
     Raises:
+        HTTPException: If the hash or hash_type is missing.
+        errors.ServerSelectionTimeoutError: If the server is not found.
```

Comment:
Not being raised anymore

Reply:
This was not resolved, why are we resolving issues that are not resolved? HTTPException is not called in this function anymore.